frontends/bmd/src/pages/superadmin_screen.test.tsx

```diff
@@ -20,14 +20,26 @@ it('the right buttons and reset calls quit', () => {
     />
   );
   screen.getByText('Reboot from USB');
+  screen.getByText('Reboot to BIOS');
   screen.getByText('Reset');
 
   // test without kiosk
   fireEvent.click(screen.getByText('Reset'));
 
   // test with kiosk
   window.kiosk = fakeKiosk();
-  fireEvent.click(screen.getByText('Reset'));
-
-  expect(window.kiosk.quit).toHaveBeenCalledTimes(1);
+  act(() => {
```

Comment:
I think `act` is unnecessary here. Was there a specific reason for adding it?

Reply:
I'm pretty sure there was a warning until I added it, but I'll check.

Reply:
huh, I was convinced it asked for it, but turns out it didn't, you are right. Removing.